.github/workflows/build_frontend.yaml

```diff
@@ -44,6 +44,21 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: '14'
-      - run: yarn install
-      - run: yarn run build
 
+      - run: |
+          yarn install
+
+      - name: Build release for production branch (conditional check)
+        if: github.ref == 'refs/heads/production'
+        run: |
+          yarn build:release
```

Comment:
is it possible to change `release` to `production` here so we don't have conflicting terminology?

Reply:
Definitely!

Reply:
Done. That's was a good callout.